src/util.js

```diff
@@ -2,3 +2,29 @@ export function convertToIntOrFallback(stringToConvert) {
   const parsed = parseInt(stringToConvert);
   return parsed ? parsed : 0;
 }
+
+export function sortByTabIndex(firstNode, secondNode) {
+  const tabIndexes = [firstNode, secondNode].map((node) =>
+    getTabIndexOfNode(node)
+  );
+  return tabIndexes
+    .map((tabIndexValue) =>
+      sanitizeTabIndexInput(tabIndexValue, Math.max(...tabIndexes))
+    )
+    .reduce((previousValue, currentValue) => previousValue - currentValue);
+}
+
+function sanitizeTabIndexInput(tabIndex, maxValue) {
```

Comment:
This function is hard to understand within this context. Can you add a comment?

Reply:
df6e776e2d2efba48d6eecb1f7e32ab56c85fd7b